tests/test_context.py

```diff
@@ -56,11 +55,50 @@ def test_async_wake_interval(device):
     """Test read-write property async_wake_interval."""
     with Context(device) as context:
         context.async_wake_interval = 42
-        assert isclose(context.async_wake_interval, 42)
+        assert context.async_wake_interval == 42
+
+
+# THIS DOES NOT WORK, PLEASE HELP!
+
+# def test_is_supported(device):
+#     with Context(device) as context:
+#         assert context == current_context()
+#         with context:
+#             MONO42 = context.is_supported("MONO", "42")
+
 
+def test_available_resamplers(device):
+    """Test available_resamplers"""
+    with Context(device) as context:
+        assert context == current_context()
+        assert len(context.available_resamplers) >= 0
```

Comment:
@McSinyx afaik, context.available_resamplers returns a list if `AL_SOFT_source_resampler` extension is supported, and returns [] if the extension is unsupported. However, I have no idea how to get those values for testing. Any idea for improvement?

Reply:
`len` never returns a negative number.

Reply:
Gentil ping on this.

Reply:
You can remove this test now, since `list.__len__` never returns negative, and even if it does it's not our problem.